src/main/java/io/openliberty/tools/common/plugins/config/ServerConfigDocument.java

```diff
@@ -432,14 +439,12 @@ private static Document getIncludeDoc(String loc) throws IOException, SAXExcepti
                 URL url = new URL(loc);
                 URLConnection connection = url.openConnection();
                 doc = parseDocument(connection.getInputStream());
+                docs.add(doc);
             }
         } else if (loc.startsWith("file:")) {
             if (isValidURL(loc)) {
                 locFile = new File(loc);
-                if (locFile.exists()) {
-                    InputStream inputStream = new FileInputStream(locFile.getCanonicalPath());
-                    doc = parseDocument(inputStream);
-                }
+                parseDocumentFromFile(locFile, docs);
```

Comment:
Do we need to consider directory case for URL?

Reply:
What does the documentation say? What does Liberty do?

Reply:
The documentation is lacking here as well. 

From inspecting Liberty code, it seems the include `location` is resolved before checking if it is a directory, so it should also handle URLs. Posed this follow-up question in slack for confirmation.

Reply:
The answer from Liberty dev team is that it does not support specifying a directory via URL in the `location` attribute for the `include` element.